bookmaru-infrastructure/src/main/kotlin/plain/bookmaru/global/security/jwt/JwtParser.kt

```
@@ -48,11 +49,10 @@ class JwtParser(
             getClaims(token)
             return true
         } catch (e: io.jsonwebtoken.security.SecurityException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)